src/handlers/predict_torch_sequence.py

```diff
@@ -0,0 +1,77 @@
+import base64
+import onnxruntime
+import torch
+import io
+import numpy as np
+import torch.nn.functional as f
+from jaqpotpy.descriptors.tokenizer import SmilesVectorizer
+from jaqpotpy.api.openapi import ModelType, PredictionRequest, PredictionResponse
+from jaqpotpy.descriptors.graph.graph_featurizer import SmilesGraphFeaturizer
+from src.helpers.utils import to_numpy
+
+
+def torch_sequence_post_handler(request: PredictionRequest) -> PredictionResponse:
+    feat_config = request.model.torch_config
+    featurizer = _load_featurizer(feat_config)
+    target_name = request.model.dependent_features[0].name
```

Comment:
Is it hard to allow multiple y target names instead of always hardcoding the 1 target_name by getting the dependent_features[0] here?